ui/app/adapters/pki/issuer.js

```diff
@@ -78,7 +61,20 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
       // To show issuer meta data tags, we have a flag called isListView and only want to
       // grab each issuer data only if there are less than 10 issuers to avoid making too many requests
       if (isListView && res.data.keys.length <= 10) {
-        return await this.getIssuerMetadata(store, type, query, res);
+        const keyInfoArray = await all(
+          res.data.keys.map((id) => {
```

Comment:
A way to shorten it (non-blocking)
```suggestion
          res.data.keys.map((id) => this.getIssuerMetadata(store, type, query, res, id))
```